pages/project-version/pdf/views/nts/components/retrospective-assessment.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import RAContent from '@asl/projects/client/constants/retrospective-assessment';
+import raApplies from '@asl/projects/client/helpers/retrospective-assessment';
```

Comment:
This was refactored away into the API data. It's probably not worth importing the module for the single line of code it contains.

https://github.com/UKHomeOffice/asl-projects/blob/master/client/helpers/retrospective-assessment.js#L2